### PR TITLE
Implementation of simple imgui integration suggestion in Issue #290

### PR DESCRIPTION
**Pull request by jstreibel**
Implementation of suggestion in Issue #290

### CODE REVIEW

File: glumpy/library/colormaps/user.glsl [hunk 1]
```diff
@@ -8,7 +8,7 @@ uniform sampler1D colormap;
 
 vec3 colormap_user(float t)
 {
-    return texture1D(colormap, t).rgb;
+    return texture(colormap, t).rgb;
```
Comment (rougier) [inline glumpy/library/colormaps/user.glsl:11]: Why this change?

**Comment by rougier** [comment]
Thanks. I think the change would be quite transparent for user and I like the glfw/glfw_imgui. Maybe we can rename the old glfw as glfw_deprecated and use your implentation as default (with name glfw/glfw_imgui). What do you think?

File: glumpy/library/colormaps/user.glsl (hunk 1 above)
Comment (jstreibel) [reply glumpy/library/colormaps/user.glsl:11]: Was not supposed to be here. I thought it would remain in my fork.

Comment (jstreibel) [reply glumpy/library/colormaps/user.glsl:11]: Still learning git, sorry.

Comment (jstreibel) [reply glumpy/library/colormaps/user.glsl:11]: It is there to correct an error (not a warning) for being deprecated since glsl 130.

**Comment by jstreibel** [comment]
Happy to help. 
The new backend uses a pretty popular and up to date glfw binding. Should make it compatible with modern stuff too.
I think its a good idea.

File: glumpy/library/colormaps/user.glsl (hunk 1 above)
Comment (jstreibel) [reply glumpy/library/colormaps/user.glsl]: So, sorry for the confusion. I think I moved it back to what it used to be.

**Comment by rougier** [comment]
Many thanks and sorry for the late answer.
